executable/FnmApp.re

```diff
@@ -170,6 +171,33 @@ let alias = {
   );
 };
 
+let default = {
+  let doc = "Alias a version as default";
+  let man = [];
+  let sdocs = Manpage.s_common_options;
+
+  let selectedVersion = {
+    let doc = "The version to be aliased as default";
```

Comment:
I can't believe you started learning Reason and Ocaml late 2018 from your given articles. There's a lot to take in, especially just trying to understand a library like lwt and cmdliner, on top of setting up the build system.

Reply:
I normally don't try to understand 100% of the things. This is why I use `pesy` here instead of pure Dune, but I also used Dune in another project where I tried to learn native OCaml/Reason.

Lwt feels a lot like JS promises, and I read a lot about functional programming and these monadic interfaces for a couple of years. This is a very big thing to learn but the `ppx_lwt` makes it easier, just like async await in JS

Cmdliner is very hard and I just read and copied from the docs until it worked! 😜  I read on the Reason Discord that there's a `ppx_deriving_cmdliner` which is interesting

Reply:
Ah thanks for the quick summary. Reread the lwt docs and they clearly make sense now. 

And I finally understand what ppx are now too. They're like decorators in JS, or method attributes in C#. LOL.

Reply:
Most of them are like decorators or method attributes, but they work at compile time, unlike JS (not sure about C# though) - including type errors, type inference and autocomplete! ❤️